flutter/practice4/lib/main.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+Future<String> str=Future.delayed(Duration(seconds: 4), () => 'Large Latte');
```

Comment:
You should not name variables to describe their type. You should describe what they mean. For example "drinkName".